explorer/syntax/parse_and_lex_context.h

```diff
@@ -55,9 +55,9 @@ class ParseAndLexContext {
 }  // namespace Carbon
 
 // Gives flex the yylex prototype we want.
-#define YY_DECL                                                         \
-  auto yylex(Carbon::Nonnull<Carbon::Arena*> arena, yyscan_t yyscanner, \
-             Carbon::ParseAndLexContext& context)                       \
+#define YY_DECL                                                             \
+  auto yylex(Carbon::Nonnull<Carbon::Arena*> /*arena*/, yyscan_t yyscanner, \
```

Comment:
This change seems odd to me, it does get used...

Reply:
I mean, if you uncomment it, Clang warns about it in one context. I didn't investigate further when commenting it out fixed the compile error.

This is declaring the *lexer* which I don't think uses this, but needs to have the same signature as the parser?